platform/etnaviv/cmds/cube_cmd.h

```diff
@@ -0,0 +1,841 @@
+
```

Comment:
May be change file extention from .h to .inc because it looks very strange header file

Reply:
Fixed it.